susemanager-utils/susemanager-sls/salt/channels/disablelocalbrokenrepos.sls

```diff
@@ -18,7 +18,7 @@ disable_repo_{{ repos_disabled.count }}:
 {% endif %}
 {% endfor %}
 {% else %}
-{% if data.get('enabled', '1') == '1' %}
+{% if data.get('enabled', True) not in [False, '0'] %}
```

Comment:
This does not need to be inverted:

```python
>>> data = {'enabled': '0'}
>>> data.get('enabled', True) in (True, '1')
False
```

`data.get('enabled', True)` returns either the string "0"/"1" for RHEL or `True`/`False` on SLE (or on RHEL if 'enabled' is not in `data`). Whatever is returned is not evaluated for truthiness (like what happens on `bool(foo)`), instead the check is if it is in the tuple (can be a list, a tuple is more idiomatic). This pattern is used to avoid chaining repeated conditions. Another way to write this would be `if data.get('enabled', '1') == '1' or data.get('enabled', True) is True`, the `in` condition is a bit cleaner.

Reply:
Yes, i found out the same. I dropped the last commit and we are now back to the ond way.